typescript/ecs/fargate-service-with-local-image/index.ts

```diff
@@ -2,7 +2,7 @@ import ec2 = require('@aws-cdk/aws-ec2');
 import ecs = require('@aws-cdk/aws-ecs');
 import ecs_patterns = require('@aws-cdk/aws-ecs-patterns');
 import cdk = require('@aws-cdk/core');
-import path = require('path');
+import path = require('path');	
```

Comment:
nit: extra space, please remove.

Reply:
f1c071b0ff8561f6edbd263bc633d51f28fc1d18 addresses this issue.